### PR TITLE
Add Jenkinsfile to trigger the re-build of the

### PR DESCRIPTION
**Pull request by hectorj2f**
related to: https://github.com/fabric8-ui/fabric8-ui-openshift-nginx/issues/7

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/groovy
+@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+def dummy
+dockerTemplate{
+  dockerNode{
```
Comment (rawlingsj) [inline Jenkinsfile]: should that be `goTemplate`?

Comment (rawlingsj) [inline Jenkinsfile]: You only need `dockerNode{..}` if you only need a docker build pod.  If you want others like golang / maven etc then we combine them using fooTemplate

Comment (hectorj2f) [reply Jenkinsfile]: why ? there isn't anything related to go here... Maybe it requires another template but we just run nginx inside.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    if (env.BRANCH_NAME.startsWith('PR-')) {
+      goCI{
```
Comment (rawlingsj) [inline Jenkinsfile]: This will run CI for a typical golang project so I dont think you'll want this.

**Review by rawlingsj — APPROVED** [review]
The dummy file looks good.  Lets merge and check the job is created.

**Pull request merged**